src/foundry/client/pixi/core/interaction/render-flags.d.mts

```diff
@@ -35,12 +35,20 @@ declare class RenderFlagObject {
  * @privateRemarks Values are marked as optional here based on use, foundry docs incomplete
  * @internal
  */
-type _RenderFlags<Flags> = InexactPartial<{
-  /** Activating this flag also sets these flags to true */
-  propagate: Array<keyof Flags>;
+type _RenderFlags<_Flags extends AnyObject> = InexactPartial<{
+  /**
+   * Activating this flag also sets these flags to true
+   * @privateRemarks Without including the list of valid flags as a type param on the mixin somehow, trying to make this
```

Comment:
This sounds like it's because the mixin results in `AnyObject`; `keyof RemoveIndexSignatures<AnyObject>` is `never`.